src/ekka.appup.src

```diff
@@ -1,7 +1,11 @@
 %% -*- mode: erlang -*-
-{"0.8.1.4",
-  [{"0.8.1.3",
-    [{load_module,ekka_autocluster,brutal_purge,soft_purge,[]},
+{"0.8.1.5",
+  [{"0.8.1.4",
+    [{load_module,ekka_node_monitor,brutal_purge,soft_purge,[]}
```

Comment:
For my own understanding: is the change motivating this `ekka_node_monitor` module coming from [this PR](https://github.com/emqx/ekka/pull/135)?

More broadly: how can I know which modules should I put in an appup? :slightly_smiling_face:

Reply:
Yes, it's related to that PR. For the second question, there is a script: https://github.com/emqx/emqx/blob/main-v4.3/scripts/update_appup.escript